src/shared/components/mutationTable/column/CosmicColumnFormatter.tsx

```diff
@@ -113,10 +118,66 @@ export default class CosmicColumnFormatter {
             cosmicData
         );
 
+        const columns: Column<CosmicMutation>[] = [
+            {
+                name: 'COSMIC ID',
+                render: (d: CosmicMutation) =>
+                    d.proteinChange === data[0].proteinChange ? (
+                        <b>
+                            <a
+                                href={`http://cancer.sanger.ac.uk/cosmic/mutation/overview?id=${d.cosmicMutationId}`}
+                                target="_blank"
+                            >
+                                {d.cosmicMutationId}
+                            </a>
+                        </b>
+                    ) : (
+                        <span>
+                            <a
+                                href={`http://cancer.sanger.ac.uk/cosmic/mutation/overview?id=${d.cosmicMutationId}`}
+                                target="_blank"
+                            >
+                                {d.cosmicMutationId}
+                            </a>
+                        </span>
+                    ),
+                sortBy: (d: CosmicMutation) => d.cosmicMutationId,
+            },
+            {
+                name: 'Protein Change',
+                render: (d: CosmicMutation) =>
+                    d.proteinChange === data[0].proteinChange ? (
+                        <b>{d.proteinChange}</b>
+                    ) : (
+                        <span>{d.proteinChange}</span>
+                    ),
+                sortBy: (d: CosmicMutation) =>
+                    calcProteinChangeSortValue(d.proteinChange),
+            },
+            {
+                name: 'Occurrence',
+                render: (d: CosmicMutation) =>
+                    d.proteinChange === data[0].proteinChange ? (
+                        <b>{d.count}</b>
+                    ) : (
+                        <span>{d.count}</span>
+                    ),
+                sortBy: (d: CosmicMutation) => d.count,
+            },
+        ];
+
         let value: number = -1;
         let display: string = '';
         let overlay: (() => JSX.Element) | null = null;
         let content: JSX.Element;
+        let lastUpdatedText: JSX.Element;
+
+        if (
+            getServerConfig().app_name === 'public-portal' ||
+            getServerConfig().app_name === 'mskcc-portal'
```

Comment:
@gblaih do you see a more uniform way of doing this in the code somewhere? I e.g. saw this:

https://github.com/cBioPortal/cbioportal-frontend/blob/dd4d311eb36138b649e2f44f4e0199d19a587f84/src/shared/components/query/QueryContainer.tsx#L125

Maybe that should have a `isMSKPortal` function too?

Reply:
https://github.com/cBioPortal/cbioportal-frontend/blob/ee88b3ef5145206539a332470db99537c5cac666/src/AppStore.ts#L65